cvat/apps/webhooks/event_type.py

```diff
@@ -13,12 +13,13 @@ class Events:
     RESOURCES = {
         "project": ["create", "update", "delete"],
         "task": ["create", "update", "delete"],
+        "job": ["create", "update", "delete"],
+        "label": ["create", "update", "delete"],
```

Comment:
Users choose what kind of events they want to receive, they can just don't choose `label` events in case they don't need them. But we cannot imagine all possible use cases, maybe for some users such an event could be useful.

Reply:
I'm mainly worried that it might clog up our webhook queue if we start sending out requests for each label change. Imagine someone creating a project with 1000 labels.

Reply:
Yes, but the same argument could be applied to all other objects "Imagine someone creates 1000 projects".

Reply:
You can't create 1000 projects all at once, whereas you _can_ create a project with 1000 labels. Moreover, there are real-world datasets with 1000 labels (e.g., ImageNet).

Reply:
Okay, I deleted `label` event